target/linux/ramips/mt7621/base-files/etc/board.d/01_leds

```diff
@@ -29,6 +29,11 @@ gehua,ghl-r-001)
 dlink,dir-860l-b1)
 	ucidef_set_led_netdev "wan" "wan" "$boardname:green:net" "eth0.2"
 	;;
+dlink,dir-878-a1)
+	ucidef_set_led_netdev "wan" "wan" "$boardname:green:net" "eth0.2"
+	ucidef_set_led_netdev "wlan2g" "WiFi 2.4GHz" "mt76-phy0" "wlan0"
+	ucidef_set_led_netdev "wlan5g" "WiFi 5GHz" "mt76-phy1" "wlan1"
```

Comment:
WLAN LEDs should not be set to netdev (can be dropped) because 1. driver can handle them automatically 2. it stops working if interface name changes e.g. multi-APs.
WAN LED should be set to switch trigger because of reason 2.

Reply:
ok I removed the wlan currently on the current driver the led are on some times when wifi is not
changed wan to switch & made green 1Ghz & orange 10/100Mhz